tests/integration/main.go

```diff
@@ -19,6 +19,8 @@ var includeMultiConsumer = flag.Bool("include-multi-consumer", false, "include m
 var parallel = flag.Bool("parallel", false, "run all tests in parallel")
 var localSdkPath = flag.String("local-sdk-path", "",
 	"path of a local sdk version to build and reference in integration tests")
+var useGaia = flag.Bool("use-gaia", false, "use gaia instead of ICS provider app")
+var gaiaTag = flag.String("gaia-tag", "", "gaia tag to use - default is latest")
```

Comment:
Could the latest tag come from v8? I could also see it not being worth worrying about v8 since it hopefully wont be around for a long time

Reply:
As you said, v8 will not be around for long (v9 upgrade is on March 15th). I propose we don't spend time on it at this point